src/components/Datagrid/usePreventHistoryBack.ts

```diff
@@ -0,0 +1,30 @@
+import throttle from "lodash/throttle";
+import React from "react";
+
```

Comment:
This has a potential to be reusable, we could add jsdoc-style comment how to use it and put it in `src/hooks`